### DISCUSSION TITLE
疑问 createRoot为何不复用之前创建的root，每次都是重新createRoot？

### DISCUSSION THREAD

**Discussion by Lemonreds** [umijs/umi#11906] [General]
<img width="1076" alt="image" src="https://github.com/umijs/umi/assets/23014238/8643639f-6b7e-49ba-8c8c-201e443b2084">

**Comment by fz6m** [comment]
这个渲染函数只会在应用启动的时候执行一次，你认为应该如何复用

**Reply by Lemonreds** [reply]
了解了，不存在多次调用 renderClient 的情况么。
